src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DeadlineTest {
+
+    @Test
+    void toDataFormatTest() {
+        Deadline d = new Deadline("return book","2/12/2019", "1800");
+        assertTrue(d.toDataFormat().equals("D | 0 | return book | 2/12/2019, 1800"));
+    }
+    /*
```

Comment:
It's always better to include as many tests as you can, rather than leavinng them commented out, don't you agree?